docs/Arrays.rst

```diff
@@ -250,7 +250,7 @@ buffer; it will be copied upon its first mutation:
 
 .. _deferred:
 
-Deferrred Checking for Forced Downcasts
+Deferred Checking for Forced Downcasts
 .......................................
```

Comment:
The number of '.' needs to match the length of the header on the preceding line.